gpytorch/lazy/lazy_tensor.py

```diff
@@ -158,7 +158,7 @@ def _getitem(self, *indices):
             which does some additional work. Calling this method directly is discouraged.
 
         Args:
-            :attr:`indices` (tuple of int, slice, or LongTensor):
+            :attr:`indices` (tuple of `int`s, `slice`s, or `LongTensor`s):
```

Comment:
I think this is wrong, I don't think `_get_indices` can ever get slices. slices are handled by `__getitem__` / `_getitem`.